src/home.js

```diff
@@ -0,0 +1,21 @@
+const home = () => {
+  const homeClasses = 'img-background-home';
+
+  const homeContent = `
+        <div class="container">
```

Comment:
when the page is loaded (Use `document.createElement` and `document.appendChild` 
Check my comment....